### PR TITLE
disable uninstalling South

### PR DESCRIPTION
**Pull request by boney-bun**
South is no longer installed on django

### CODE REVIEW

**Review by timlinux — COMMENTED** [review]
This PR seems to mix in other things too - lets get your Makefile stuff merged separately first

File: deployment/docker/Dockerfile-dev [hunk 1]
```diff
@@ -27,7 +27,7 @@ RUN echo "export VISIBLE=now" >> /etc/profile
 
 ADD REQUIREMENTS-dev.txt /REQUIREMENTS-dev.txt
 RUN pip install -r /REQUIREMENTS-dev.txt
-RUN pip uninstall -y South
+# RUN pip uninstall -y South
```
Comment (ismailsunni) [inline deployment/docker/Dockerfile-dev]: Perhaps just remove it, less cluttered

**Comment by boney-bun** [comment]
Hi @timlinux @meomancer @ismailsunni 
I've pushed a clean commit now.

**Review by ismailsunni — APPROVED** [review]
LGTM

**Comment by boney-bun** [comment]
thanks @ismailsunni 
i'll merge them

**Pull request merged**